UNITTESTS/stubs/connectivity/AT_CellularDevice_stub.cpp

```diff
@@ -56,6 +56,7 @@ ATHandler *AT_CellularDevice::get_at_handler()
 
 CellularContext *create_context(const char *apn)
 {
+    return NULL;
```

Comment:
This is cpp, can we not return `nullptr` instead of `NULL`? `nullptr` is less ambiguous than `NULL` in that it always returns a pointer type and can't acquire the type `int`, which `NULL` can as it's literally just 0.

Reply:
Fixed